akka-kernel/src/main/dist/bin/akka-cluster

```diff
@@ -33,9 +33,13 @@ function get() {
 }
 
 function ensureNodeIsRunningAndAvailable {
-    REPLY=$(get Available) # redirects STDERR to STDOUT before capturing it
+    REPLY=$(get Available 2>&1) # redirects STDERR to STDOUT before capturing it
     if [[ "$REPLY" != *true ]]; then
-        echo "Akka cluster node is not available on $HOST, due to $REPLY"
+        if [[ "$REPLY" == *"Cannot convert result to a string." ]]; then
```

Comment:
is this check robust? Or can it mask errors that has nothing to do with MBean not aviailable?